### ISSUE TITLE
Alt text for messages

### ISSUE THREAD

**Issue by Wallby** [vector-im/element-meta#2896]
#### What would you like to do?
I'd like to be able to add alt text to GIFs, images, videos and link thumbnails/previews.

#### Why would you like to do it?
I am in a chat group on WhatsApp currently and there is a person who is visually impaired. Sometimes a joke gets made using a GIF, and it would be nice to add the alt text in a way that that user can access it but it doesn't have to be implemented via some work around such as a caption or another message, such that if a person forgets to add an alt text they can go back to the message and add it later.

#### How would you like to achieve it?
Being able to add the alt text during upload and also being able to modify a message if it was forgotten, because accessibility is a thing people are likely to forget, so it should be amendable.

### Have you considered any alternatives?

I saw #378, but as I said I don't want to use a workaround. Native support would be ideal.
